CacheMoney/frontend/src/components/PhoneDisplay.js

```diff
@@ -0,0 +1,17 @@
+import React, { useState } from "react";
+
+function PhoneDisplay(){
+    //const [account, setAccount] = useState({});
+
+    return (
+        <div className="phone-outer-container">
+            <div className="content-container">
+                <p>Phone</p>
+                <hr></hr>
+
+            </div>
+        </div>
+    )
+}
+
+export default PhoneDisplay;
```

Comment:
As far as I can tell we do not store a users phone number. Do we want to get rid of this?